### ISSUE TITLE
Transcode Audio only transcodes on host, do video transcodes on other nodes.

### ISSUE THREAD

**Issue by sachk** [UnicornTranscoder/UnicornTranscoder#57]
Hello, I was wondering whether it's possible to handle audio transcoding on the plex host, but do video transcoding on the other nodes. Running a node on the plex host is fine.

**Comment by Maxou44** [comment]
Not possible, but you could install a transcoder node on your main server :)

**Comment by sachk** [comment]
Yes, I could do that, but my main server isn't powerful enough for video transcoding. So, I want it to  handle only audio transcoding jobs. Perhaps an option to have a separate list of nodes that are used when only audio transcoding is needed would be appropriate?

**Comment by Maxou44** [comment]
In this case why you don't change your main server for a less powerfull server ?

Are you talking about music transcode or video stream with bad audio format ?

**Comment by sachk** [comment]
I'm talking about video stream with bad audio format. My setup has a pretty powerful cheap OpenVZ box with terrible peering, and a cheap KVM box with very good peering and network settings that make it work far better for the 200ms 30+Mbps streams I require. They peer very well to eachother, so I want the OpenVZ box to be used just for transcoding video, and everything else left to the KVM box to avoid wasting the limited bandwidth.

**Comment by Maxou44** [comment]
It's a really specific issue... Maybe fix your peering issue could be the best solution 😁

In all cases if we support that, we will never bundle it in the load blancer, you will have to setup a transcoder on your main server.

Maybe we could improve the score detection based on ffmpeg parameters to priorize servers (detect copy streams). Not on our roadmap at the moment.

**Comment by drouarb** [comment]
Maybe we could introduce transcoder capabilities, something in transcoder config that would looks like
```json
features: {
  video: 'direct_play',
  audio: 'all'
}
```
This could allow the LoadBalancer to filter transcoders on features available for the transcoder

**Comment by Maxou44** [comment]
It could be nice if we have flags like `direct_play`, `direct_stream`, `transcode` or `all` for video and audio streams?

**Comment by sachk** [comment]
I never said that we should implement it in the load balancer, in my first message I said that it'd be fine if it was implemented as part of the transcoder.

**Comment by sachk** [comment]
The transcoder capabilities feature sounds like a good idea to me.
